crates/forge/tests/it/spec.rs

```diff
@@ -1,14 +1,10 @@
 //! Integration tests for EVM specifications.
 
 use crate::{config::*, test_helpers::TEST_DATA_DEFAULT};
-use foundry_evm::revm::primitives::SpecId;
 use foundry_test_utils::Filter;
 
 #[tokio::test(flavor = "multi_thread")]
 async fn test_shanghai_compat() {
     let filter = Filter::new("", "ShanghaiCompat", ".*spec");
-    TestConfig::with_filter(TEST_DATA_DEFAULT.runner(), filter)
-        .evm_spec(SpecId::SHANGHAI)
-        .run()
-        .await;
+    TestConfig::with_filter(TEST_DATA_DEFAULT.runner(), filter).run().await;
 }
```

Comment:
Is this correct? It appears to test specifically against Shanghai compatibility - not sure of the context, perhaps we can generalize the name into `test_compat()`?

Reply:
good point, I kept it the way it was, with deafult paris and making sure shanghai compatible, see https://github.com/foundry-rs/foundry/pull/9131/commits/702c270be810314cd81e0a12d19a52e9a934cff0